site/static/js/main.js

```diff
@@ -531,3 +531,15 @@ document.addEventListener('s72loaded', function(event) {
   let app = event.detail.app;
   documentReady(app);
 });
+
+// Last resort iframe title insertion for WCAG compliance.
+new MutationObserver(function(mutations, observer) {
+  mutations.forEach(function(mutation) {
+    mutation.addedNodes.forEach(function(node) {
+      if (node.nodeName !== 'IFRAME') return;
+      if (!node.getAttribute('aria-hidden') || node.getAttribute('title')) return;
```

Comment:
This logic could be at risk if it has `aria-hidden="false"` with a `title` - it will override the given title.

Reply:
Ah good point!